src/tools/perf/api/libperf.h

```diff
@@ -98,6 +98,29 @@ enum {
 };
 
 
+#define FOREACH_UCP_PERF_DAEMON_AM_ID \
+    /* Host to daemon communication */  \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_INIT,     0) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_REQ,      1) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_SEND_ACK, 2) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_RECV_ACK, 3) \
+    X(UCP_PERF_DAEMON_AM_ID_HOST_FIN,      4) \
```

Comment:
btw, where do we send INIT and FIN?

Reply:
we send them from libperf.c, which will be in the 3rd batch commit

Reply:
It's possible, but if I bring those changes in this PR, they will come with all the remaining daemon part (allocation of memory handles, export memory) which is another 250-300LOC, and basically we don't need 3rd commit in this case. This is also an option, I let you decide

I was happy with current split, because 2nd commit concludes on daemon process side, and all the remaining changes are in host process side different files.

Reply:
ok